README.md

```diff
@@ -19,16 +19,16 @@ To start a server in Docker container you should:
     * `ACCOUNTS_VAULT_BASIC_PATH` - path to cloudstack accounts' secrets in vault ("secret/cs/accounts/" by default). Last "/" is required
     * `VM_VAULT_BASIC_PATH` - path to vms' secrets in vault ("secret/cs/vms/" by default). Last "/" is required
     * `TAG_NAME_PREFIX` - prefix for token name tag
-    * `KAFKA_SERVER_LIST` - list of kafka urls
+    * `KAFKA_ENDPOINTS` - list of kafka endpoints
     * `KAFKA_TOPIC` - kafka topic containing cloudstack events
-    * `ZOOKEEPER_URL` - zookeeper url
+    * `ZOOKEEPER_ENDPOINTS` - list of zookeeper endpoints, which is separated by commas
```

Comment:
list of zookeeper endpoints separated by a comma

Reply:
ok